src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-console */
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/users.service');
+
+const get = async(req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  try {
+    const allExpenses = await expensesService.getAllExpenses(
+      userId, categories, from, to);
+
+    if (!allExpenses) {
+      res.status(404).send('Not Found: The specified entity does not exist');
+    }
+
+    res.send(allExpenses);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send('Internal Server Error');
+  }
+};
+
+const getOne = async(req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    res.status(400).send('Bad Request: Missing required parameter');
```

Comment:
make sure to use `return` keyword here in order to prevent rest of the function from running.